server/src/main/java/io/spine/server/commandbus/CommandBus.java

```diff
@@ -92,6 +95,18 @@
     @LazyInit
     private @MonotonicNonNull CommandValidator commandValidator;
 
+    /**
+     * An observer which processes the {@linkplain io.spine.base.ThrowableMessage rejections}
+     * thrown by the bus {@linkplain BusFilter filters}.
+     *
+     * <p>Is NO-OP at bus creation. Once an {@link EventBus} is
```

Comment:
`When the bus is first created, this observer does nothing. Once an {@link EventBus} is ...`